### PR TITLE
fix: Need to linkify during Markdown rendering

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: ietf/utils/markdown.py [hunk 1]
```diff
@@ -27,23 +37,14 @@ def markdown(text):
                 python_markdown.markdown(
                     text,
                     extensions=[
+                        # TODO: discuss which extensions we want to enable, see
```
Comment (rjsparks) [inline ietf/utils/markdown.py]: We did discuss this at the time this went in. I'm ok refreshing the discussion, but what's here isn't whimsical.

**Comment by rjsparks** [comment]
Is there a test that exercises this config change to make sure it does what we want, and would let us know if something later breaks it?

File: ietf/utils/markdown.py (hunk 1 above)
Comment (larseggert) [reply ietf/utils/markdown.py]: Want me to remove the comment?

**Comment by larseggert** [comment]
Let me think about that.

**Comment by larseggert** [comment]
@rjsparks added a test.

**Review by rjsparks — APPROVED** [review]

**Pull request merged**